synthesizer/tests/utilities/tests/program_test.rs

```diff
@@ -94,45 +94,63 @@ impl ExpectedTest for ProgramTest {
             .clone();
 
         // Parse the remainder of the test file into a program.
-        let program = Program::<CurrentNetwork>::from_str(&source[first_comment_start + 2 + end_first_comment + 2..])
-            .expect("Failed to parse program.");
+        let programs = source[first_comment_start + 2 + end_first_comment + 2..]
+            .split("/////////////////////////////////////////////////")
```

Comment:
just to double-check; this is the separator?

Reply:
Yes